README.md

```diff
@@ -10,14 +10,123 @@
+
+## Rails File Structure Overview
+
+Since you will be working with this file structure on a daily basis, it is very important to understand and become familiar with the file system. Below is a breakdown for each directory:
+
+* app – contains the models, views, and controllers, along with the the rest of the core functionality of the application. This is the one directory where you can make a change and not have to restart the Rails server. The majority of your time will be spent working in this directory. In addition to the full MVC structure, this directory also contains the full Rails asset pipeline directories, such as: css files, javascripts, images, fonts, etc.
+
+* bin – some built-in Rails tasks that you most likely will never have to work with.
+
+* config – the config directory manages a number of settings that control the default behavior, including: the environment settings, a set of modules that are initialized when the application starts, the ability to set language values, the application settings, the database settings, the application routes, and lastly the secret key base.
+
+* db – within the db directory you will find the database schema file that lists the database table, their columns and the column’s associated data types. The db directory also contains the seeds.rb file, which lets you create some data that can be utilized in the application. This is a great way to quickly integrate data in the application without having to manually add records through a web form element.
+
+* lib – while many developers could build full applications without ever entering the lib directory, you will discover that it can be incredibly helpful. The lib/tasks directory is where custom rake tasks are created. You have already used a built-in rake task when you ran the database creation and migration tasks; however, creating custom rake tasks can be very helpful and sometimes necessary. An example of a rake task I have created recently was a rake task that I had the server run in the background that called an outside API and synced the API data into the application’s database.
+
+* log – within the log directory you will discover the application logs. This can be helpful for debugging purposes, but for production application I will typically use a outside service since they can offer more advanced services such as querying and alerts.
+
+* public – this directory contains some of the custom error pages, such as 404 errors, along with the robots.txt file which will let developers control how search engines index the application on the web.
+
+* test – if you did not bypass the Minitest installation, Rails will install the test directory; if you added the option of –skip-test-unit this test directory will not be included.
```

Comment:
did we tell them to bypass the minitest installation? Need a bit more explanation on that